packages/cloudflare/templates/defaults/wrangler.toml

```diff
@@ -0,0 +1,7 @@
+main = ".open-next/worker.js"
+name = "app-name"
+
+compatibility_date = "2024-12-30"
+compatibility_flags = ["nodejs_compat"]
+
+assets = { directory = ".open-next/assets", binding = "ASSETS" }
```

Comment:
Maybe add a commented out kv binding for the cache

Reply:
done 👍